src/cookbook/networking/update-data.md

```diff
@@ -26,14 +26,11 @@ This recipe uses the following steps:
 ## 1. Add the `http` package
 
 To install the `http` package,
-add it to the dependencies section
-of the `pubspec.yaml` file.
-You can find the latest version of the
-[`http` package][] on pub.dev.
-
-```yaml
-dependencies:
-  http: <latest_version>
+add it to the dependencies section of the `pubspec.yaml` file
+by running `flutter pub add`:
```

Comment:
**suggestion:** Write in active voice per [GSG](https://developers.google.com/style/voice).
```suggestion
run `flutter pub add`.
This command adds `http` to the `dependencies` section of the `pubspec.yaml` file:
```